test/com/esotericsoftware/kryo/SerializationCompatTest.java

```diff
@@ -109,9 +109,24 @@ public void testDefaultSerializers() throws Exception {
                 EXPECTED_DEFAULT_SERIALIZER_COUNT, defaultSerializers.size());
     }
 
-    public void testStandard () throws Exception {
+    public void testStandardOptGenerics () throws Exception {
+   	  kryo.getFieldSerializerConfig().setOptimizedGenerics(true);
         runTests(
-                "standard",
+                "standard-opt-generics",
```

Comment:
The existing versioned files `test/resources/*-standard.ser` should be `git mv`'ed to `test/resources/*-standard-opt-generics.ser`.